atomic_reactor/utils/operator.py

```diff
@@ -17,6 +17,13 @@
 from atomic_reactor.util import ImageName, chain_get
 
 
+__all__ = [
```

Comment:
Do we need this? Somebody may not expect this and have hard times with imports

Reply:
True, I mostly just wanted to express that `NamedPullspec` and its subclasses are not really a public part of the module, but it's not that important. I'll remove this.